evalml/tests/automl_tests/test_automl.py

```diff
@@ -1064,7 +1063,6 @@ def test_add_to_rankings(
         y_train=y,
         problem_type="binary",
         max_iterations=1,
-        allowed_component_graphs=dummy_classifier_linear_component_graph,
```

Comment:
`allowed_component_graphs` plays no role in the validity of this test, so removing :)